pkg/controller/podupdater/pod_updater_controller.go

```diff
@@ -0,0 +1,502 @@
+/*
```

Comment:
how about rename file name `pod_updater_controller.go` to `daemon_pod_updater_controller.go`?